backend/backend/speech_client.py

```diff
@@ -0,0 +1,4 @@
+from google.cloud import speech
+from .settings import credentials
```

Comment:
This is a minor but I've already been using absolute imports rather than relative imports, e.g. `backend.settings` vs `.settings`. It's usually helpful as they're more readable and less ambiguous, especially when moving around code
Doesn't need to be fixed but something to note for the future